data/json/items/ammo.json

```diff
@@ -4410,6 +4410,31 @@
         "dispersion" : 0,
         "recoil" : 0,
         "count" : 1
+    },
+    {   "type" : "AMMO",
+        "id" : "gas_fungicidal",
+        "category" : "chems",
+        "price" : 1200,
+        "name" : "sprayable fungicide",
+        "symbol" : "=",
+        "color" : "yellow",
+        "description" : "Fungicide enhanced with sulfur to create sprayable anti-fungal ammo for the chemical thrower.",
+        "material" : "hydrocarbons",
```

Comment:
You should probably give it some other material. "null" should be fine.
Hydrocarbons is for stuff that fuels fires really well.